google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQuery.java

```diff
@@ -774,6 +864,41 @@ public int hashCode() {
    */
   Table update(TableInfo tableInfo, TableOption... options);
 
+  /**
+   * Updates model information.
+   *
+   * <p>Example of updating a model by changing its description.
+   *
+   * <pre>{@code
+   * String datasetName = "my_dataset_name";
+   * String modelName = "my_model_name";
+   * String newDescription = "new_description";
+   * Model beforeModel = bigquery.getModel(datasetName, modelName);
+   * ModelInfo modelInfo = beforeModel.toBuilder()
+   *     .setDescription(newDescription)
+   *     .build();
+   * Model afterModel = bigquery.update(moddelInfo);
```

Comment:
Typo: `moddel`

Note: We have a little program at https://github.com/googleapis/google-cloud-java/blob/master/utilities/snippets.go for copying code samples into JavaDocs, if you want to try that out.

Reply:
Fixed the typo.  Inclined to add the snippets in a followup and redo the instrumentation then, unless you want this change to get even bigger.